modules/KIWIBoot.pm

```diff
@@ -4608,6 +4608,12 @@ sub setupBootLoaderConfiguration {
         # console (console) or the serial terminal (serial)
         my $console_mode = 'graphics';
 
+        # Set serial console mode for ec2* firmwares. This is a hack for
+        # Azure images in order to avoid new attributes in XML schema.
+        if ($type->{firmware} =~ m/ec2/) {
```

Comment:
Also needs the condition for $type->{format} eq 'vhdfixed'

Then we catch both Azure and EC2